service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java

```diff
@@ -84,9 +87,18 @@ public void populateCase(@PathVariable("caseId") Long caseId, @RequestBody Map<S
         }
     }
 
+    @GetMapping("/testing-support/case/{caseId}/lastEvent")
+    public AuditEvent getLastEvent(@PathVariable("caseId") String caseId) {
+        return coreCaseDataApiV2.getAuditEvents(requestData.authorisation(), authToken.generate(), true, caseId)
```

Comment:
Is there a reason we are saying the request is experimental here? In #2537 we saying the request isn't experimental

Reply:
changed to false to match other invocation. But from ccd pov value does not matter, it is just required marker header